com.unity.netcode.gameobjects/Runtime/Serialization/NetworkBehaviourReference.cs

```diff
@@ -5,7 +5,7 @@ namespace Unity.Netcode
 {
     /// <summary>
     /// A helper struct for serializing <see cref="NetworkBehaviour"/>s over the network. Can be used in RPCs and <see cref="NetworkVariable{T}"/>.
-    /// Note: network ids get recycled by the NetworkManager after a while. So a reference pointing to
+    /// Note: network IDs get recycled by the NetworkManager after a while. So a reference pointing to
```

Comment:
Network IDs get recycled after a while, so a reference pointing to.....?????

Also can we make any recommendations to users to avoid running into this problem?

Reply:
Since NetworkBehaviourIds get recycled by the NetworkManager, keeping the NetworkBehaviourReference around for extended periods of time could return a different NetworkBehaviour for the assigned NetworkBehaviourId. It is recommended to only use NetworkBehaviourReference to acquire the NetworkBehaviour from the NetworkBehaviourReference when received from an RPC or custom message and not use it for any long term use.